content/applications/general/email_communication/email_servers.rst

```diff
@@ -231,14 +156,16 @@ first outgoing email server (sorted by priority).
    example, if there are two email servers, one with a priority of `10` and the other with a
    priority of `20`, then the email server with a priority of `10` is used first.
 
+.. _email_communication/mass_mails:
+
 Set up different dedicated servers for transactional and mass mails
-===================================================================
+-------------------------------------------------------------------
 
 In Odoo a separate Mail Transfer Agent (MTA) server can be used for transactional emails and mass
```

Comment:
Mail Transfer Agent (MTA) is not used anywhere else. I would stick to the generic term "mail server" or "outgoing mail server".

Reply:
Noted, thanks.